audit-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/audit/rest/HealthActuatorContractTest.java

```diff
@@ -45,31 +44,24 @@
 @ActiveProfiles({"akka-test"})
 class HealthActuatorContractTest {
 
-    @Autowired
-    private TestRestTemplate restTemplate;
-    @Autowired
-    private Map<String, AuditService> serviceMap;
-
     @Test
-     void testContextLoads() {
-       assertThat(serviceMap)
-               .hasSize(3)
-               .containsKeys("simple", "stroom", "logger");
-        assertThat(restTemplate).isNotNull();
+    void testContextLoads(@Autowired final Map<String, AuditService> serviceMap) {
+        assertThat(serviceMap).containsOnlyKeys("simple", "stroom", "logger");
```

Comment:
```suggestion
        assertThat(serviceMap)
        .as("Check the Audit Service loads three different implementations of the Audit Service") 
        .containsOnlyKeys("simple", "stroom", "logger");
```
or some sort of similar description

Reply:
Updated